### PR TITLE
#274: update metarpheus version (closes #274)

### LINKED ISSUE #274: update metarpheus version

### PR DESCRIPTION
**Pull request by giogonzo**
Closes [#2521075](https://buildo.kaiten.io/space/[object Object]/card/2521075)

⚠️ The line above was added during the automatic migration of all github project issues to Kaiten. The old GH issue link is kept here for reference (it is now deprecated, continue follow the original issue on Kaiten).

Closes #274

### tests performed

> A Test Plan is used to show what you tested to make sure your code works fine. You should write here all that you did to test, and provide some results of your testing.

> These results can be screenshots, query results, or even just pointers to unit tests that successfully passed.

### tests not performed (domain coverage)

> At times not everything can be tested, and writing what hasn't been tested is just as important as writing what has been tested.

> An example of partial test is a field displaying 4 possible values. If 3 values are tested, with screenshots, and 1 is not, then it should be mentioned here.

### CODE REVIEW

File: test/metarpheus.test.ts [hunk 1]
```diff
@@ -1,32 +1,37 @@
...
   .join('/');
-const metarpheusOutPath = path.resolve(templateDir, metarpheusDir);
+const metarpheusOutPath = path.resolve(testAppDir, metarpheusDir);
 
 afterAll(() => {
   rimraf.sync(metarpheusOutPath);
 });
 
-describe('metarpheus', () => {
+fdescribe('metarpheus', () => {
```
Comment (gabro) [inline test/metarpheus.test.ts]: what’s fdescribe?

File: test/metarpheus.test.ts [hunk 2]
```diff
@@ -1,32 +1,37 @@
...
+        `cd ${testAppDir}`,
+        './node_modules/.bin/scriptoni metarpheus --metarpheusConfig metarpheus.config.test.js'
+      ]).then(() => {
         expect(fs.existsSync(apiPath)).toBeTruthy();
         expect(fs.existsSync(modelsPath)).toBeTruthy();
 
         expect(fs.readFileSync(apiPath, { encoding: 'utf8' })).toMatchSnapshot();
         expect(fs.readFileSync(modelsPath, { encoding: 'utf8' })).toMatchSnapshot();
+      }, err => {
+        console.log((err as any).message) return Promise.reject(err)
```
Comment (gabro) [inline test/metarpheus.test.ts]: add a newline before `return`?

File: test/globalSetup.js [hunk 3]
```diff
@@ -13,7 +13,11 @@ module.exports = () => {
   return runCommands([
     `cd ${scriptoniDir}`,
     'yarn build',
-    `cd ${templateDir}`,
-    'yarn --frozen-lockfile'
+    `cd ${__dirname}`,
+    'npx @buildo/create-bento-app@2.0.0-1 test-app',
```
Comment (gabro) [inline test/globalSetup.js]: mo-no-re-po! 🙌

**Review by gabro — APPROVED** [review]
👍 just a couple of nits

File: test/metarpheus.test.ts (hunk 1 above)
Comment (giogonzo) [reply test/metarpheus.test.ts]: good catch, it's a leftover (it should `f`ocus on the single suite of tests, but I think it only has effects inside the single file so doesn't do nothing here)

File: test/metarpheus.test.ts (hunk 2 above)
Comment (giogonzo) [reply test/metarpheus.test.ts]: another leftover... I was "debugging"

File: test/globalSetup.js (hunk 3 above)
Comment (giogonzo) [reply test/globalSetup.js]: lo so lo so, con le cose di questi giorni cose mergiate siamo già a -2 librerie, prossimo step è quello

**Pull request merged**